### ISSUE TITLE
About the bootstrapped cross entropy

### ISSUE THREAD

**Issue by zhouweii234** [hkchengrex/STCN#63]
If the bootstrap cross entropy loss is directly added to another loss loss function (for example, mean squared error Loss or IoU loss), will these two functions still work normally?

**Comment by hkchengrex** [comment]
Yes. You can just consider them as two separate gradient streams. You need to beware of scaling though since one loss might be insignificant compared to the other.
